src/pydvl/utils/caching/__init__.py

```diff
@@ -84,5 +84,11 @@
 from .base import *
 from .config import *
 from .disk import *
-from .memcached import *
 from .memory import *
+
+try:
+    from .memcached import *
+
+    PYMEMCACHE_INSTALLED = True
```

Comment:
This variable is not used anymore, right?

Reply:
Correct, I removed it